src/create_disk.sh

```diff
@@ -114,11 +114,9 @@ case "$arch" in
     x86_64)
         EFIPN=2
         set -- -Z $disk \
```

Comment:
Minor: at this point, we might as well drop the `set --` method and just call `sgdisk` directly. That way it also matches the way it's done on the other arches.